src/delivery-options/data/prices/getLowestPriceFromFormConfig.js

```diff
@@ -17,9 +17,9 @@ export function getLowestPriceFromFormConfig(config, carrierName = null, configB
     .map((carrierName) => {
       const parentEnabled = configBus.isEnabled(config, null, carrierName);
```

Comment:
config.options zou dezelfde variabele kunnen gebruiken

Reply:
Nee, de eerste parameter is anders

Reply:
ah je hebt gelijk. ik last over de param heen. 👍🏻